components/DependencyPropertyGenerator/CommunityToolkit.DependencyPropertyGenerator.SourceGenerators/Constants/WellKnownTypeNames.cs

```diff
@@ -0,0 +1,52 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace CommunityToolkit.GeneratedDependencyProperty.Constants;
+
+/// <summary>
+/// The well known names for types used by source generators and analyzers.
+/// </summary>
+internal static class WellKnownTypeNames
+{
+    /// <summary>
+    /// The fully qualified type name for the <c>[GeneratedDependencyProperty]</c> type.
+    /// </summary>
+    public const string GeneratedDependencyPropertyAttribute = "CommunityToolkit.WinUI.GeneratedDependencyPropertyAttribute";
+
+    /// <summary>
+    /// The fully qualified type name for the <c>GeneratedDependencyProperty</c> type.
+    /// </summary>
+    public const string GeneratedDependencyProperty = "CommunityToolkit.WinUI.GeneratedDependencyProperty";
+
+    /// <summary>
+    /// The fully qualified type name for the <c>XAML</c> namespace.
+    /// </summary>
+    public const string XamlNamespace =
+#if WINDOWS_UWP
+        "Windows.UI.Xaml";
+
+#else
+        "Microsoft.UI.Xaml";
+#endif
```

Comment:
@Sergio0694 would it be worth trying to see how this could work with WPF as well here? Theoretically it'd just be a different namescape here too, eh? `System.Windows` https://learn.microsoft.com/dotnet/api/system.windows.dependencyproperty?view=windowsdesktop-8.0 - or would we not be able to distinguish WinUI from WPF with the .NET TFM the same way unless we had a third package?

Reply:
Correct, yeah we'd need to ship a separate package just for WPF, also we have no tests, etc. 😅

Reply:
Yeah, that's what I thought and a larger discussion I've been thinking about for some other stuff too. Bubbled it up elsewhere to at least think about for 2025 as we move forward.